Strategies/components/everlend/EverlendDeposit.tsx

```diff
@@ -185,7 +185,7 @@ const EverlendDeposit = ({
       </div>
       <Input
         type="number"
-        onChange={(e) => setAmount(e.target.value)}
```

Comment:
typechecker complained about coerced values. Maybe I should have used @ts-ignore instead.